### ISSUE TITLE
When prices are entered exclusive tax, and are set to be displayed inclusive tax on the product and checkout page - Firefox and possibly Edge and IE still display the price exclusive tax.

### ISSUE THREAD

**Issue by grigorijurasov** [woocommerce/woocommerce-gateway-stripe#564]
#### Affected ticket(s) 

983429 - zd-woothemes

#### What I expected

I was expecting to see prices inclusive tax in Firefox on shop, cart and checkout pages.

#### What happened instead

When I follow the steps below, I see that in Chrome, prices in the shop, cart and at the checkout are displayed including tax, however, in Firefox (customer also reported IE and Edge, haven't tested those) prices are displayed excluding tax on a shop, cart and checkout pages.

#### Steps to reproduce the issue

1. In WooCommerce >> Settings >> Tax >> Tax options, choose "No, I will enter prices exclusive of tax":
![screenshot](http://cld.wthms.co/uPPNoB.jpg) **Image Link:** http://cld.wthms.co/uPPNoB
2. "Display prices in the shop" and "Display prices during cart and checkout" set to "Including tax".
![screenshot](http://cld.wthms.co/RcwxfH.jpg) **Image Link:** http://cld.wthms.co/RcwxfH
3. Add 10% tax rate:
![screenshot](http://cld.wthms.co/D0A5qO.jpg) **Image Link:** http://cld.wthms.co/D0A5qO
4. Open a product in Firefox.
5. If Stripe 4.0.7 is enabled - prices wil be shown exlusive tax on shop, cart and checkout pages, and it will not be possible to checkout. With Stripe disabled, this behaviour disappears. WooCommerce 3.3.3

Adding High priority since with this setup (which I believe isn't so rare), when a customer uses one of the mentioned browsers - shop is not functional.

-------------------

- [ ] Issue assigned to next milestone.
- [ ] Issue assigned a priority (will be assessed by maintainers).

**Comment by roykho** [comment]
So you're saying this is caused by Stripe? If you disable Stripe, this issue immediately goes away?

**Comment by grigorijurasov** [comment]
That's correct. I disabled Stripe and immediately the price was displayed correctly on all pages.

**Comment by grigorijurasov** [comment]
Had it set to "Customer shipping address", used the address in Austria.
![screenshot](http://cld.wthms.co/qjlEGR.jpg) **Image Link:** http://cld.wthms.co/qjlEGR

**Comment by roykho** [comment]
Ok so I am not seeing issues with this. This is what I see on product page ![](http://cld.wthms.co/AFMux+) and this is cart page ![](http://cld.wthms.co/EKqqrd+)
